larpandoracontent/LArVertex/CandidateVertexCreationAlgorithm.h

```diff
@@ -148,6 +148,10 @@ class CandidateVertexCreationAlgorithm : public pandora::Algorithm
     float                   m_extrapolationStepSize;            ///< The extrapolation step size in cm
     float                   m_maxCrossingSeparationSquared;     ///< The separation (squared) between spacepoints below which a crossing can be identified
     float                   m_minNearbyCrossingDistanceSquared; ///< The minimum allowed distance between identified crossing positions
+
+    bool                    m_reducedCandidates;                ///< Whether to reduce the number of candidates
+    float                   m_par1;
```

Comment:
Please add short descriptive comments here for par1 and par2, which will be renamed (as per above).